### PR TITLE
Pop-up windows

### PR DESCRIPTION
**Pull request by Yorsyboy**
## Description
In this PR, I implemented the pop-up for the project section for both the mobile and web view.

### General requirements

- [x] There are no linter errors.
- [x] Followed the **Github flow**.
- [x] The documentation is professional.

Thank you for taking the time to review this pull request. :pray:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -64,128 +65,7 @@ <h3 class="intro2">LET’S CONNECT</h3>
...
-                        <li class="skill set2">css</li>
-                        <li class="skill set3">javascript</li>
-                    </ul>
-                    <div class="button-click">
-                        <button type="button" class="btn">See Project</button>
-                    </div>
-                </div>
-            </div>
-
-            <div class="projects">
```
Comment (micheaol) [inline index.html:128]: Please ensure that the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, margins, transparent background, width, and height). Also please implement the same on the mobile version.


Your Implementation | Figma Design
:---:|:---:
image | image

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @Yorsyboy , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (Whoistolu) [reply index.html:128]: - [ ] [REQUIRED] Just as the last reviewer recommended, kindly ensure that the popup properly follows the design for both desktop and mobile versions (e.g., font, colors, margins, transparent background, width, and height). 
Currently, you have vertical scroll bars you need to remove on the desktop popup, and the popup for the mobile version is not well aligned.
### Your design (Desktop)
Screenshot from 2022-04-06 12-45-35
Screenshot from 2022-04-06 12-45-40
### Your design (Mobile)
Screenshot from 2022-04-06 12-49-20

Here is what your Popup is supposed to look like
### Figma Template
Screenshot from 2022-04-06 13-10-20

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @Yorsyboy 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Whoistolu) [reply index.html:128]: Hello, @Yorsyboy I understand your point. Please try to resize the sample popup from the template. You can see that it has its length longer than the width but your design has its width wider, consider imitating the design for the template ad all its fonts, font size, width, and height. I am sure if you adhere to this you will get the desired outcome. Well done 👍🏽 and Happy coding!!!

File: index.html [hunk 2]
```diff
@@ -64,128 +65,7 @@ <h3 class="intro2">LET’S CONNECT</h3>
...
-                <div class="img-web">
-                    <img class="" src="images/project3.png" alt="project1">
-                </div>
-                <div class="project-description">
-                    <h1 class="title">Tonic</h1>
-                    <div class="role">
-                        <div class="role-name">CANOPY</div>
-                        <div class="role-dot"></div>
-                        <div class="role-job">Back End Dev</div>
```
Comment (MeqdamAlqudah) [inline index.html:137]: - [ ] As the last review said please make sure that your design matches the Figma design 
add  a blurred background to your pop window as in the Figma design, you said that the content of the project will be hidden so I don't see this as an issue hide the content of the portfolio when the user opens the popup and make it visible it when the user closes the popup

| Your design | Figma design |
| ------------- | --------------- |
|  image | image |

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @Yorsyboy , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.





 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Yorsyboy) [reply index.html:137]: @MeqdamAlqudah Hi, Thanks for the suggestion about the blurry part, once I enlarge the image this is what happens

Screenshot (63)

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Yorsyboy, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**